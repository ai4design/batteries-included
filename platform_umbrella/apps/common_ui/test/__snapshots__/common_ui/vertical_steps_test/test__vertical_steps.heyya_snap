<ul class="steps steps-vertical">
  <li class="step step-secondary">
    Step 0
  </li><li class="step step-secondary">
    Step 1
  </li><li class="step">
    Step 2
  </li>
</ul>