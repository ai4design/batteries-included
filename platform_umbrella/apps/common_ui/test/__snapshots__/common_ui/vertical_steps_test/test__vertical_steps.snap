"<ul class=\"steps steps-vertical\">\n  <li class=\"step step-secondary\">\n    Step 0\n  </li><li class=\"step step-secondary\">\n    Step 1\n  </li><li class=\"step\">\n    Step 2\n  </li>\n</ul>"